src/Components/Clock.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+type Props = {
+  setHasClock: (hasClockProps: boolean) => void;
+  clockName: string;
+};
+
+type State = {
+  time: Date;
+};
+
+function timeActualy(time: Date): string {
```

Comment:
The function name must begin with a verb.
And why, if you name the type for the argument `Date`, then you name the argument itself `time`?
```suggestion
function getTimeFromDate(currentDate: Date): string {
```